rhoas-cli/README.adoc

```diff
@@ -140,14 +132,10 @@ $ sudo dnf install -y https://github.com/redhat-developer/app-services-cli/relea
 
```

Comment:
Line 133 in the original unedited file: In the RPM installation section, remove “This example installs rhoas:”.

Reply:
Done